test/e2e/storage/vsphere/vsphere_volume_vsan_policy.go

```diff
@@ -309,7 +307,9 @@ func invokeInvalidPolicyTestNeg(client clientset.Interface, namespace string, sc
 	return fmt.Errorf("Failure message: %+q", eventList.Items[0].Message)
 }
 
-func invokeStaleDummyVMTestWithStoragePolicy(client clientset.Interface, masterNode string, namespace string, clusterName string, scParameters map[string]string) {
+// invokeStaleDummyVMTestWithStoragePolicy assumes control plane node(aka. master node) is present on the datacenter specified in the workspace section of vsphere.conf file.
```

Comment:
we should remove references of the word "master" from the code base, so here we can remove "(aka. master node)"

Reply:
OK, done.